prepared_test.go

```diff
@@ -0,0 +1,28 @@
+package useragent_test
+
+import (
+	"context"
+	"os/exec"
+	"os/signal"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// This test is done weird way, as we need access to the embed module information
```

Comment:
**suggestion**: Test an unexported function that takes a `debug.BuildInfo` and returns the useragent instead of spawning a new process.

Reply:
yeah, I originally thought about passing it to the exported function, but with unexported one it makes more sense. Thanks!